hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/util/ExpressionUtils.java

```diff
@@ -167,6 +167,8 @@ public static Object getValueFromLiteral(ValueLiteralExpression expr) {
       case SMALLINT:
       case INTEGER:
         return expr.getValueAs(Integer.class).orElse(null);
+      case BIGINT:
+        return expr.getValueAs(Long.class).orElse(null);
       case FLOAT:
         return expr.getValueAs(Float.class).orElse(null);
```

Comment:
Can we add some tests for it, when the sql literal could have BIGINT type ?

Reply:
I have added a unit test, please help me to check, thank you.